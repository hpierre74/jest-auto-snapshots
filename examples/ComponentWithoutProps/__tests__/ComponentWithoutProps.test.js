import ComponentWithoutProps from '../ComponentWithoutProps';
import snap from 'jest-auto-snapshots';

snap(ComponentWithoutProps);
