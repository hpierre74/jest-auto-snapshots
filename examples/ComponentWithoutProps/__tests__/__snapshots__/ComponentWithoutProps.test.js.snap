// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > ComponentWithoutProps Matches snapshot when rendered 1`] = `
<span>
  No Props Needed
</span>
`;
